catalog/ui/catalog-ui-search/src/main/webapp/js/model/Workspace.collection.js

```diff
@@ -122,7 +125,8 @@ module.exports = Backbone.Collection.extend({
         var queryForWorkspace = new Query.Model({
             title: 'Example Location',
```

Comment:
Similarly, should we be using the user's defaults for searches here?

Reply:
I'm not sure that we need user defaults in the example searches, though. I'd expect those to be static across all users regardless of their search settings.